### PR TITLE
Refactor timeout notification for use with connection lost message

### PR DESCRIPTION
**Pull request by HelloUniverseNo31415**
# Checklist:

Related work items: #200 

Preview url: [N/A]

- [ ] Tests have been added for new functionality (Need puppeteer tests, havent started them yet as I need help)
- [x] Work has been reviewed by other team members
- [x] Work has been reviewed by @craigayre or @eclennett

### CODE REVIEW

File: src/components/NotificationBar/NotificationBar.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      <div
+        className={
+          'notification-bar--notification--' +
+          (this.state.show ? 'show' : 'hide')
+        }
+        id="notificationBar">
+        <div className="notification-bar--info">
+          <div className="notification-bar--alert">{this.props.mainText}</div>
+          <div className="notification-bar--timer">
+            {this.props.autoActionWord} in{' ' + this.state.countdown}s
```
Comment (eclennett) [inline src/components/NotificationBar/NotificationBar.js]: could just have `} in {this.`

File: src/components/NotificationBar/NotificationBar.scss [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  text-align: right;
+  font-size: 30px;
+  padding-top: 24px;
+  padding-right: 24px;
+  color: #47bed8;
+}
+
+.notification-bar--alert {
+  padding-top: 5px;
+  text-align: left;
```
Comment (eclennett) [inline src/components/NotificationBar/NotificationBar.scss]: is text-align: left not automatic

File: src/components/App/App.js [hunk 3]
```diff
@@ -1,72 +1,90 @@
...
+  resetTimeoutTimer = () => {
+    this.setState({showTimer: false});
+    clearTimeout(this.timer);
+    this.timer = setTimeout(
+      this.showTimeoutMessage,
+      WAIT_BEFORE_DISPLAY * 1000
+    );
+  };
+
+  timeout = () => {
```
Comment (eclennett) [inline src/components/App/App.js]: this doesn't look like a timeout to me...?

File: src/components/NotificationBar/NotificationBar.scss (hunk 2 above)
Comment (HelloUniverseNo31415) [reply src/components/NotificationBar/NotificationBar.scss]: In this case it defaults to center if not specified

File: src/components/NotificationBar/NotificationBar.js [hunk 4]
```diff
@@ -45,7 +45,7 @@ class NotificationBar extends Component {
         <div className="notification-bar--info">
           <div className="notification-bar--alert">{this.props.mainText}</div>
           <div className="notification-bar--timer">
-            {this.props.autoActionWord + ' in' + ' ' + this.state.countdown}s
+            {this.props.autoActionWord + ' in ' + this.state.countdown}s
```
Comment (eclennett) [inline src/components/NotificationBar/NotificationBar.js]: its in a div so you can just type I actually meant its best to do
`{this.props.autoActionWord} in {this.state.countdown}s`

**Pull request merged**